.github/workflows/simulation-1.2.yml

```diff
@@ -98,3 +98,39 @@ jobs:
     - name: Codecov
       run: |
         bash <(curl -s https://codecov.io/bash) -x "${{ matrix.compiler.gcov }}" -Z
+
+  packet-verification:
```

Comment:
nit: 
```suggestion
  low-power-packet-verification:
```

Reply:
I think we can put all 1.2 packet verifications here. That may need us to re-name some of the test file names. I'm planning to it to organize our tests better.

Reply:
Most other 1.2 packet verification would involves the Backbone link so they will be put the `thread-cert/backbone` directory as implemented by #5489.

Reply:
Ok. Let's add the prefix.

Reply:
Thanks